atlascope/core/management/commands/populate.py

```diff
@@ -56,32 +46,15 @@ def expand_references(obj, model):
             remote_model = found_many_to_many.remote_field.model
             if remote_model == Dataset:
                 many_to_many_values[field_name] = [remote_model.objects.get(name=x) for x in value]
-            elif remote_model == User:
-                many_to_many_values[field_name] = [remote_model.objects.get(email=x) for x in value]
+            # elif remote_model == User:
+                # many_to_many_values[field_name] = [remote_model.objects.get(email=x) for x in value]
     for field_name in many_to_many_values.keys():
         del obj[field_name]
-    for field_name, permission in {
-        'investigators': 'change_investigation',
-        'observers': 'view_investigation',
-    }.items():
-        if field_name in obj:
-            permissions[permission] = [
-                User.objects.get(username=username) for username in obj[field_name]
-            ]
-            del obj[field_name]
-    return obj, many_to_many_values, files_to_save, permissions
+    return obj, many_to_many_values, files_to_save
 
 
-@click.option('--password', type=click.STRING, help='password to apply to all users')
```

Comment:
We need to update lines 8-9 in the README; the command invocation has changed and `createsuperuser` is now required if you want to use the admin console.

Reply:
See 40f93d89f0845ea254775bca52d8d414f3bbe3a9.